x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/persistence/InferenceInternalIndex.java

```diff
@@ -95,9 +98,6 @@ private static void addInferenceDocFields(XContentBuilder builder) throws IOExce
             .startObject(TrainedModelConfig.CREATE_TIME.getPreferredName())
             .field(TYPE, DATE)
             .endObject()
-            .startObject(TrainedModelConfig.DEFINITION.getPreferredName())
```

Comment:
Don't we need to disable indexing for the `definition` somewhere else?

Reply:
By default, we don't dynamically index new fields (dynamic = false)